bigquery/snippets/snippet.go

```diff
@@ -1325,6 +1370,18 @@ func printTableInfo(client *bigquery.Client, w io.Writer, datasetID, tableID str
 	return nil
 }
 
+func printModelInfo(client *bigquery.Client, w io.Writer, datasetID, modelID string) error {
+	ctx := context.Background()
+	// [START bigquery_get_model]
+	meta, err := client.Dataset(datasetID).Model(modelID).Metadata(ctx)
+	if err != nil {
+		return err
+	}
+	fmt.Fprintf(w, "Got model '%s' with friendly name '%s'\n", modelID, meta.Name)
```

Comment:
Can use `%q` to automatically quote strings.

Reply:
Done